shell/src/main/java/alluxio/cli/fs/command/PersistCommand.java

```diff
@@ -56,6 +58,15 @@
           .desc("Number of concurrent persist operations, default: " + DEFAULT_PARALLELISM)
           .required(false)
           .build();
+  private static final int DEFAULT_TIMEOUT = 20 * Constants.MINUTE_MS;
+  private static final Option TIMEOUT_OPTION =
+      Option.builder("t")
+          .longOpt("timeout")
+          .argName("timeout milliseconds")
+          .numberOfArgs(1)
+          .desc("Time in milliseconds for a single file persist to time out: " + DEFAULT_TIMEOUT)
```

Comment:
is it clear here that the number is the default timeout?

Reply:
good catch